app/views/layouts/application.html.erb

```diff
@@ -19,10 +19,10 @@
   <link rel="shortcut icon" href="<%= asset_path('meta/favicon-32x32.png') %>" sizes="32x32">
 
   <%= tag.link href: asset_path('application.css'), rel: 'stylesheet' %>
-
   <!--[if lt IE 9]>
 	  <%= javascript_include_tag 'html5shiv' %>
   <![endif]-->
+  <%= yield :head %>
```

Comment:
This allows pages to insert more markup into the <head> tag if they wish. It's how the PPM Estimator's print-specific CSS gets included.